### PR TITLE
small updates

### PR DESCRIPTION
**Pull request by realthreeseconds**

### CODE REVIEW

File: src/main/java/forceitembattle/commands/CommandStats.java [hunk 1]
```diff
@@ -43,7 +37,24 @@ public boolean onCommand(CommandSender commandSender, Command command, String s,
...
+
+        if(player.isOp()) {
+            if(strings.length == 2) {
+                if (strings[0].equalsIgnoreCase("reset")) {
+                    if (!this.forceItemBattle.getStatsManager().playerExists(strings[1])) {
+                        player.sendMessage("§e" + strings[1] + " §cdoes not exist");
+                        return false;
+                    }
+                    this.forceItemBattle.getStatsManager().resetStats(strings[1]);
+                    player.sendMessage("§aSuccessfully resetted stats of §e" + strings[1]);
```
Comment (FireBladeHunter) [inline src/main/java/forceitembattle/commands/CommandStats.java:52]: "resetted" existiert nicht 
-> "reset"

Comment (realthreeseconds) [reply src/main/java/forceitembattle/commands/CommandStats.java]: mb, fixed

**Review by realthreeseconds — COMMENTED** [review]
👍 good qol

**Review by McPlayHD — APPROVED** [review]

**Comment by McPlayHD** [comment]
I didn't know you could craft suspicious stew lol

**Pull request merged**